pkg/config/event_watcher.go

```diff
@@ -0,0 +1,137 @@
+// Copyright 2021 The PipeCD Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package config
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+)
+
+type EventWatcherSpec struct {
+	Events []EventWatcherEvent `json:"events"`
+}
+
+// EventWatcherEvent defines which file will be replaced when the given event happened.
+type EventWatcherEvent struct {
+	// The event name. The combination of this name and labels must be unique within a project.
```

Comment:
Do we need this statement: "The combination of this name and labels must be unique within a project."
Because piped is only handling the events of its project.

Reply:
Sorry for the confusion, as you feel this shouldn't be here.

I just mean that "if you use the same name across repositories, they affect each other. So you have to make that combination unique if you want to explicitly distinguish them".